Pulsarc/UI/Screens/Gameplay/GameplayEngineView.cs

```diff
@@ -221,27 +217,25 @@ public override void Draw(GameTime gameTime)
                 return;
 
             // Don't bother drawing the background if dim is 100%
-            if (background.Dimmed && background.DimTexture.Opacity != 1f || !background.Dimmed)
-                background.Draw();
+            if (_background.Dimmed && _background.DimTexture.Opacity != 1f || !_background.Dimmed)
+                _background.Draw();
 
-            crosshair.Draw();
+            _crosshair.Draw();
             DrawArcs();
 
             foreach (TextDisplayElementFixedSize tdef in uiElements)
                 tdef.Draw();
 
-            judgeBox.Draw();
-            accMeter.Draw();
+            _judgeBox.Draw();
+            _accMeter.Draw();
         }
 
         private void DrawArcs()
         {
-            bool skip;
-
             // Go through each key
             for (int i = 0; i < GetGameplayEngine().KeyCount; i++)
             {
-                skip = false;
+                var skip = false;
```

Comment:
Don't create a new variable each time in loop, move variable initialization out of the loop.